[2024-10-12T18:14:51.025+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-12T18:14:51.036+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first-dag.task2 scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2024-10-12T18:14:51.045+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first-dag.task2 scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2024-10-12T18:14:51.045+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-12T18:14:51.090+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task2> on 2022-02-03 00:00:00+00:00
[2024-10-12T18:14:51.097+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first-dag', 'task2', 'scheduled__2022-02-03T00:00:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpc9oj_azp']
[2024-10-12T18:14:51.098+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1447) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-12T18:14:51.100+0000] {standard_task_runner.py:72} INFO - Started process 1461 to run task
[2024-10-12T18:14:51.100+0000] {standard_task_runner.py:105} INFO - Job 283: Subtask task2
[2024-10-12T18:14:51.139+0000] {task_command.py:467} INFO - Running <TaskInstance: first-dag.task2 scheduled__2022-02-03T00:00:00+00:00 [running]> on host 50fe842129eb
[2024-10-12T18:14:51.200+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='james' AIRFLOW_CTX_DAG_ID='first-dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2022-02-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-03T00:00:00+00:00'
[2024-10-12T18:14:51.201+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-12T18:14:51.221+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-12T18:14:51.222+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Running task2"']
[2024-10-12T18:14:51.232+0000] {subprocess.py:86} INFO - Output:
[2024-10-12T18:14:51.234+0000] {subprocess.py:93} INFO - Running task2
[2024-10-12T18:14:51.234+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-12T18:14:51.256+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-12T18:14:51.260+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first-dag, task_id=task2, run_id=scheduled__2022-02-03T00:00:00+00:00, execution_date=20220203T000000, start_date=20241012T181451, end_date=20241012T181451
[2024-10-12T18:14:51.321+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-12T18:14:51.336+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-12T18:14:51.339+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
